phpstan.neon.dist

```diff
@@ -46,6 +47,8 @@ parameters:
         - '#Unsafe usage of new static\(\)\.#'
         - '#Constructor of class rex_form_.*_element has an unused parameter \$tag.#'
         - '#^Offset .* on .* always exists#'
+        - '#Attribute class (?:AllowDynamicProperties|JetBrains\\PhpStorm\\Deprecated) does not exist\.#'
+        - '#Parameter \#2 \$algo of function password_.+ expects int, string given\.#'
```

Comment:
Ist das ein phpstan bug?

Reply:
Es ist denke ich kein Bug, sondern einfach ein Problem durch die Konstellation Check per php8, und target version php 7.3.
Die Konstanten wie `PASSWORD_DEFAULT` sind erst seit 7.4 strings, waren vorher ints. Somit verlangten in 7.3 auch die Funktionen ints als Parameter. phpstan erkennt die Konstante aber als string (da mit php 8 ausgeführt).